docs/ecosystem/contracts/faq.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: "Frequently Asked Questions"
+title: "Ink! FAQ"
 ---
 
 This page will answer a number of common questions you may have when starting to build smart contracts for Substrate.
 
-## ink!
+## Ink! FAQ
```

Comment:
@robbepop wants to keep the brand `ink!` with a lowercase `i`.

So it is symmetrical with !